### PR TITLE
fix: links on celestia-architecture README

### PR DESCRIPTION
**Pull request by rootulp**
Supersedes https://github.com/celestiaorg/celestia-core/pull/795

### CODE REVIEW

File: docs/celestia-architecture/README.md [hunk 1]
```diff
@@ -12,19 +12,19 @@ Hence, we do not provide any extensive documentation here.
...
+- <https://github.com/tendermint/tendermint/tree/master/docs/>
+- <https://github.com/tendermint/spec>
 
 Reading these will give you a lot of background and context on Tendermint which will also help you understand how celestia-core and [celestia-app](https://github.com/celestiaorg/celestia-app) interact with each other.
 
 ## Celestia
 
 As mentioned above, celestia-core aims to be more focused on the Celestia use-case than vanilla Tendermint.
 Moving forward we might provide a clear overview on the changes we incorporated.
-For now, we refer to the Celestia specific [ADRs](./adr) in this repository as well as to the Celestia specification:
```
Comment (rootulp) [inline docs/celestia-architecture/README.md:25]: Remove broken link

File: docs/celestia-architecture/README.md [hunk 2]
```diff
@@ -57,6 +57,6 @@ To start a new ADR, you can use this template: [adr-template.md](./adr-template.
 - [ADR 002: Sampling erasure coded Block chunks](./adr-002-ipld-da-sampling.md)
 - [ADR 003: Retrieving Application messages](./adr-003-application-data-retrieval.md)
 - [ADR 004: Data Availability Sampling Light Client](./adr-004-mvp-light-client.md)
-- [ADR 005: Decouple BlockID and PartSetHeader](./adr-005-decouple-blockid-and-partsetheader)
-- [ADR 006: Row Propagation](./adr-006-row-propagation)
-- [ADR 007: Minimal Changes to Tendermint](./adr-007-minimal-changes-to-tendermint)
\ No newline at end of file
+- [ADR 005: Decouple BlockID and PartSetHeader](./adr-005-decouple-blockid-and-partsetheader.md)
```
Comment (rootulp) [inline docs/celestia-architecture/README.md:60]: Fixes link

**Review by adlerjohn — APPROVED** [review]

**Comment by evan-forbes** [comment]
can we add ADR 008 while we're at it?

**Review by evan-forbes — APPROVED** [review]
thanks!

**Pull request merged**